plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergFileOperations.java

```diff
@@ -234,7 +233,6 @@ public void testInsert()
                         .add(new FileOperation(STATS, "InputFile.newStream"))
                         .add(new FileOperation(SNAPSHOT, "OutputFile.create"))
                         .add(new FileOperation(MANIFEST, "OutputFile.create"))
-                        .add(new FileOperation(MANIFEST, "InputFile.newStream"))
```

Comment:
Maybe we should outline in the commit comment that the manifest list corresponding to the table snapshot is not read anymore now during writes.